client/src/index.tsx

```diff
@@ -5,7 +5,7 @@ import React from "react";
 import ReactDOM from "react-dom";
 import { BrowserRouter as Router } from "react-router-dom";
 
-import App from "./components/App/App";
+import App from "./pages/App/App";
```

Comment:
moved `components` folder to `pages` and using `components` folder for reusable small components.

Reply:
Nice